Python/Bollywood.py

```diff
@@ -0,0 +1,43 @@
+import random
+movies = ('LAGAAN', 'HIGHWAY', 'THOR', 'DANGAL', 'NEWTON','ANEK','APNE','CHHICHHORE','WAR','GULAAL','DHOOM','MALANG', 'STAR WARS')
```

Comment:
Add comments to explain what the program is all about

Reply:
okay sir